src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
```

Comment:
While the function `convertToObject` is correctly named and describes its purpose, it would be beneficial to include a JSDoc comment above the function declaration to describe its parameters, return type, and purpose for better code documentation and readability.